docs/zh/week11/11-2.md

```diff
@@ -0,0 +1,526 @@
+---
+lang-ref: ch.11-2
+lecturer: Yann LeCun
+title: Loss Functions (cont.) and Loss Functions for Energy Based Models
+authors: Charles Brillo-Sonnino, Shizhan Gong, Natalie Frank, Yunan Hu
+date: 13 April 2020
```

Comment:
Use `Apr` instead of `April`.
`translator:` and `translation-date:` are missing.

Reply:
Thanks! Just added!